rules/BUILD.java_runtime.tpl

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
BUILD.java.tpl

Not sure if we might add more java stuff to it, e.g. java_toolchain()

Reply:
fixed